src/components/pages/DownloadPage.jsx

```diff
@@ -1,31 +1,36 @@
 import Instructions from 'components/elements/Instructions'
-import { DOWNLOAD_URL } from 'constants/urls'
+import InstructionsUninstall from 'components/elements/InstructionsUninstall'
```

Comment:
If this is a page, where this is the only content that will be present, we should add a h1/h2 tag or something similar.
At least an h1, with sr-only (screen reader only) property for accessibility, if you run axe-dev-tools against the page it should spit out the various minor issues to fix

Reply:
Every page needs at least 1 h1 tag, but you can make it only for screen readers using sr-only, so that it doesn't affect your h3's down at the bottom (they'd change to h2's, FWIW).

Reply:
Ah...okay, that makes sense! I changed the h3's to h1's because that came up as part of the axe-dev-tools scan. Now there's h1 tags on the page, I think we're good to go now.